dotnet/src/Extism.Sdk/Plugin.cs

```diff
@@ -71,20 +71,31 @@ public bool FunctionExists(string name)
     /// <param name="functionName">Name of the function in the plugin to invoke.</param>
     /// <param name="data">A buffer to provide as input to the function.</param>
     /// <returns>The exit code of the function.</returns>
```

Comment:
TODO: update this docstring, maybe mention it can throw -- not sure what is common way of expressing this in c#

Reply:
done